### ISSUE TITLE
Security audit failure due to outdated chokidar version

### ISSUE THREAD

**Issue by DullReferenceException** [babel/babel#10210]
## Bug Report

**Current Behavior**
`npm audit` reports some vulnerabilities in `mixin-deep` which is a deep sub-dependency of `chokidar`. Sample:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ mixin-deep                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.0.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @babel/cli [dev]                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @babel/cli > chokidar > readdirp > micromatch > snapdragon > │
│               │ base > mixin-deep                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1013                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Expected behavior/code**
`@babel/cli` should not have dependencies that fail security audits

**Environment**
- Babel version(s): 6.5.0
- Node/npm version: node 10.15.3, npm 6.10.1
- OS: macOS 10.14.5
- Monorepo: no
- How you are using Babel: cli

**Comment by JLHwung** [comment]
This issue duplicate #10176 

In short, this issue only affects `babel-cli --watch`, see the comment [here](https://github.com/babel/babel/issues/10176#issuecomment-510787801).

**Comment by DullReferenceException** [comment]
I don't think it's a duplicate, because #10176 is for `babel-cli`, not `@babel/cli`.

**Comment by nicolo-ribaudo** [comment]
(Question: how do I enable npm warnings?)

<details>

```
dev/babel-bugs/security 
➜ npm init -f
npm WARN using --force I sure hope you know what you are doing.
Wrote to /home/nicolo/Documenti/dev/babel-bugs/security/package.json:

{
  "name": "security",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}



dev/babel-bugs/security is 📦 v1.0.0 via ⬢ v11.14.0 
➜ npm i -D @babel/core
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN security@1.0.0 No description
npm WARN security@1.0.0 No repository field.

+ @babel/core@7.5.4
added 36 packages from 55 contributors and audited 200 packages in 3.532s
found 0 vulnerabilities


dev/babel-bugs/security is 📦 v1.0.0 via ⬢ v11.14.0 took 4s 
➜ npm i -D @babel/cli 
npm WARN security@1.0.0 No description
npm WARN security@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ @babel/cli@7.5.0
added 149 packages from 70 contributors and audited 2376 packages in 5.777s
found 0 vulnerabilities


dev/babel-bugs/security is 📦 v1.0.0 via ⬢ v11.14.0 took 6s 
➜ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
found 0 vulnerabilities
 in 2376 scanned packages

dev/babel-bugs/security is 📦 v1.0.0 via ⬢ v11.14.0 took 2s
```

</details>

**Comment by nicolo-ribaudo** [comment]
Oh ok, it has been fixed in `mixin-deep` v1.3.2: https://www.npmjs.com/advisories/1013

which is the version which gets installed:
```
dev/babel-bugs/security is 📦 v1.0.0 via ⬢ v11.14.0 
➜ npm ls mixin-deep
security@1.0.0 /home/nicolo/Documenti/dev/babel-bugs/security
└─┬ @babel/cli@7.5.0
  └─┬ chokidar@2.1.6
    └─┬ braces@2.3.2
      └─┬ snapdragon@0.8.2
        └─┬ base@0.11.2
          └── mixin-deep@1.3.2 
```

I'd say just run `npm update mixin-deep`

**Comment by DullReferenceException** [comment]
Excellent; that fix must have been recently released. `npm audit fix` fixes it now.
